stacks/azure/application/centraldashboard/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../common/centraldashboard/overlays/stacks
```

Comment:
Why not import centraldashboard directly?

Reply:
Good question. I didn't modify this -- it was like this before. I am planning to talk to some people to refactor this nonsense, but didn't want to alter it now.

Reply:
got you. For short term, please remove `/metadata` link, https://github.com/kubeflow/manifests/commit/caf2c9bcc738843e7ebd6cd52e28dc0ebd0257ea

underneath server has been removed.